### DISCUSSION TITLE
How to set 60fps?

### DISCUSSION THREAD

**Discussion by Moonbesk** [rustdesk/rustdesk#3779] [Q&A]
How to set 60fps?

**Comment by aogg** [comment]
+1

**Comment by youhers** [comment]
Reduce the resolution

**Comment by aogg** [comment]
We should cancel the compression algorithm, or use low compression to improve the fps by increasing the speed of the intranet.

**Comment by 21pages** [comment]
try #5264 if using windows

**Reply by aogg** [reply]
The method of 60fps with low configuration (without gpu,1 core)
